test/test_rocrand_kernel_sobol32.cpp

```diff
@@ -318,13 +382,26 @@ TEST(rocrand_kernel_sobol32, rocrand_log_normal)
         hipMemcpy(m_vector, h_directions, sizeof(unsigned int) * 8 * 32, hipMemcpyHostToDevice));
     HIP_CHECK(hipDeviceSynchronize());
 
+    hipStream_t stream = 0;
+    hipGraph_t graph;
+    hipGraphExec_t graph_instance;
+    if (use_graphs)
+    {
+        // Default stream does not support hipGraph stream capture, so create a non-blocking one
+        HIP_CHECK(hipStreamCreateWithFlags(&stream, hipStreamNonBlocking));
+        graph = test_utils::createGraphHelper(stream);
+    }
+
     hipLaunchKernelGGL(
         HIP_KERNEL_NAME(rocrand_log_normal_kernel<state_type>),
         dim3(8), dim3(32), 0, 0,
         output, m_vector, output_size
     );
     HIP_CHECK(hipGetLastError());
 
```

Comment:
This kernel launch should be synced with `stream` right?
```suggestion
    hipLaunchKernelGGL(
        HIP_KERNEL_NAME(rocrand_log_normal_kernel<state_type>),
        dim3(8), dim3(32), 0, stream,
        output, m_vector, output_size
    );
    HIP_CHECK(hipGetLastError());
```

Reply:
Nice catch, thank you. This must have gotten lost when I rebased. Given @nolmoonen's point below about hipGraph testing in the kernel test files being a little overkill, I will remove all hipGraph tests from this file.